### **Multi-Omics**
General Resources for Multi-Omics

## Contents
- [Tools](#Tools)
- [Pipelines](#Pipelines)
- [Datasets](#Datasets)
- [Visualization](#Visualization)
- [Learn](#Learn)
- [RunAndTests](#RunAndTests)

## Tools
- [PMA](https://CRAN.r-project.org/package=PMA) - Performs Penalized Multivariate Analysis: a penalized matrix decomposition, sparse principal components analysis, and sparse canonical correlation analysis, described in Witten, Tibshirani and Hastie (2009) <doi:10.1093/biostatistics/kxp008> and Witten and Tibshirani (2009) Extensions of sparse canonical correlation analysis, with applications to genomic data <doi:10.2202/1544-6115.1470>.
- [gesca](https://CRAN.r-project.org/package=gesca) - Hwang - RGSCA regularized generalized structured component analysis
- [RGCCA](https://cran.r-project.org/package=RGCCA) - Tenenhaus - Regularized Generalized CCA and Sparse Generalized CCA
- [scca](https://github.com/tomwhoooo/scca_3.0) - Lee - Sparse Canonical Covariance Analysis for High-throughput Data
- [STATIS/DiSTATIS](https://github.com/HerveAbdi/DistatisR) - Abdi - structuring three-way statistical tables
- [RIMBANET](http://research.mssm.edu/integrative-network-biology/RIMBANET/RIMBANET_overview.html) - Zhu - Reconstructing Integrative Molecular Bayesian Networks
- [FactoMineR](https://cran.r-project.org/package=FactoMineR) - Abdi - MFA: multiple factor analysis
- [JIVE](https://genome.unc.edu/jive/) - Lock - joint & individual variance explained
- [pandaR](https://bioconductor.org/packages/release/bioc/html/pandaR.html) - Schlauch - Passing Attributes between Networks for Data Assimilation
- [omicade4](https://bioconductor.org/packages/omicade4) - Meng - MCIA: multiple co-interia analysis
- [STATegRa](https://bioconductor.org/packages/STATegRa) - Planell - DISCO, JIVE, & O2PLS
- [GFAsparse](https://research.cs.aalto.fi/pml/software/GFAsparse/) - Khan - group factor analysis sparse
- [CCAGFA](https://cran.r-project.org/package=CCAGFA) - Klami - Bayesian Canonical Correlation Analysis and Group Factor Analysis
- [CMF](https://cran.r-project.org/package=CMF) - Klami - collective matrix factorization
- [moGSA](https://bioconductor.org/packages/mogsa) - Meng - multi-omics gene set analysis
- [iNMF](https://github.com/yangzi4/iNMF) - Yang - integrative NMF
- [BASS](https://github.com/judyboon/BASS) - Zhao - Bayesian group factor analysis
- `imputeMFA` in [missMDA](https://cran.r-project.org/web/packages/missMDA/index.html) - Voillet - multiple imputation for multiple factor analysis (MI-MFA)
- [PLSCA](https://github.com/derekbeaton/PLSCA_Framework) - Beaton - Partial Least Square Correspondence Analysis
- [mixOmics](https://bioconductor.org/packages/mixOmics) - Rohart - various methods
- [mixedCCA](https://github.com/irinagain/mixedCCA) - Yoon - sparse CCA for data of mixed types
- [SLIDE](https://github.com/irinagain/SLIDE_Rpackage) - Gaynanova - Structural Learning and Integrative Decomposition of Multi-View Data
- [fCCAC](https://github.com/pmb59/fCCAC/) - Madrigal - functional canonical correlation analysis to evaluate covariance
- [TSKCCA](https://github.com/kosyoshida/TSKCCA) - Yoshida - Sparse kernel canonical correlation analysis
- [AJIVE](https://github.com/idc9/r_jive) - Feng - angle-based JIVE
- [MOFA](https://github.com/bioFAM/MOFA) - Argelaguet - multi-omics factor analysis
- [PCA+CCA](https://github.com/pachterlab/PCACCA/) - Brown
- [JACA](https://github.com/Pennisetum/JACA) - Zhang - Joint Association and Classification Analysis
- [pCIA](https://www.med.upenn.edu/long-lab/software.html) - Min - penalized COI
- [OmicsPLS](https://github.com/selbouhaddani/OmicsPLS) - Bouhaddani  - O2PLS implemented in R, with an alternative cross-validation scheme
- [SCCA-BC](https://github.com/pimentel/scca-bc) - Pimentel - Biclustering by sparse canonical correlation analysis
- [WON-PARAFAC](https://github.com/NKI-CCB/won-parafac) - Kim - weighted orthogonal nonnegative parallel factor analysis
- [BIDIFAC](https://github.com/lockEF/bidifac) - Park - bidimensional integrative factorization
- [SmCCNet](https://cran.r-project.org/web/packages/SmCCNet/index.html) - Shi - sparse multiple canonical correlation network analysis
- [msPLS](https://github.com/acsala/sPLSPM) - Csala - multiset sparse partial least squares path modeling
- [D-CCA](https://github.com/shu-hai/D-CCA) - Shu - Decomposition-based Canonical Correlation Analysis
- [COMBI](https://bioconductor.org/packages/combi) - Hawinkel - Compositional Omics Model-Based Integration
- [DPCCA](https://github.com/gwgundersen/dpcca) - Gundersen - Deep Probabilistic CCA
- [MEFISTO](https://biofam.github.io/MOFA2/MEFISTO.html) - Velten - spatial or temporal relationships
- [MultiPower](https://github.com/ConesaLab/MultiPower) - Tarazona - Sample size in multi-omic experiments
- [mixedCCA](https://cran.r-project.org/web/packages/mixedCCA/) - Yoon - Sparse semiparametric CCA for data of mixed types
- [ade4](https://CRAN.r-project.org/package=ade4) - Dray - Implementing the Duality Diagram for Ecologists
 - [THEME](https://github.com/ThomData/R_THEME) - Verron - THEmatic Model Exploration
 - [iCluster](https://cran.r-project.org/package=iCluster) - Shen
 - [MDI](https://warwick.ac.uk/fac/cross_fac/zeeman_institute/zeeman_research/software/) - Kirk
 - [iClusterPlus](https://bioconductor.org/packages/iClusterPlus) - Mo
 - [BCC](https://github.com/ttriche/bayesCC) - Lock - Bayesian consensus clustering
 - [iBAG](https://github.com/umich-biostatistics/iBAG) - Wang - Integrative Bayesian Analysis of Genomics
 - [SNF](http://compbio.cs.toronto.edu/SNF/SNF/Software.html) - Wang
 - [clusternomics](https://cran.r-project.org/web/packages/clusternomics/index.html) - Gabasova
 - [IBOOST](http://dlin.web.unc.edu/software/iboost/) - Wong
 - [Spectrum](https://cran.r-project.org/web/packages/Spectrum/index.html) - John
  - [INF](https://gitlab.fbk.eu/MPBA/INF) - Chierici and Bussola
  - [maui](https://github.com/BIMSBbioinfo/maui) - Ronen - Stacked VAE + clustering predictive of survival
  - [IntegrativeVAEs](https://github.com/CancerAI-CL/IntegrativeVAEs) - Simidjievski - Variational autoencoders + classification
  - [DeepProg](https://github.com/lanagarmire/DeepProg) - Poirion - DL and ML ensemble + survival prediction
  - [SHAE](https://github.com/BoevaLab/Supervised-hierarchical-autoencoders-for-cancer-survival) - Wissel - Supervised Hierarchical Autoencoder + survival prediction
  - [MolTi-DREAM](https://github.com/gilles-didier/MolTi-DREAM/) - Didier - identifying communities from multiplex networks, and annotated the obtained clusters
  - [RWR-MH](https://github.com/alberto-valdeolivas/RWR-MH) - Valdeolivas - Random walk with restart on multiplex and heterogeneous biological networks
  - [MOGAMUN](https://bioconductor.org/packages/MOGAMUN/) - Novoa-del-toro - A multi-objective genetic algorithm to find active modules in multiplex biological networks
  - [RWRF](https://github.com/Sepstar/RWRF/) - Wen -  Random Walk with Restart for multi-dimensional data Fusion
  - [cardelino](https://github.com/PMBio/cardelino) -    - gene expression states to clones (SNVs from scRNA-seq + bulk exome data)
  - [clonealign](https://github.com/kieranrcampbell/clonealign) - Campbell - gene expression states to clones (scRNA-seq + scDNA-seq (CNV))
  - [CiteFuse](https://sydneybiox.github.io/CiteFuse/) - Kim - CITE-seq data analysis
  - [CoSpar](https://cospar.readthedocs.io/) - Wang - infer dynamics by integrating state and lineage information
  - [MSFA](https://github.com/rdevito/MSFA) - De Vito - multi-study factor analysis: same features, different samples